packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -28,7 +28,10 @@ export default class Sayt extends LitElement {
    * Shows a button to allow for closing SAYT manually.
    */
   @property({ type: Boolean, reflect: true }) showCloseButton = false;
-
+  /**
+   * Customizes the number of characters entered in searchbox before opening SAYT.
+   */
+  @property({ type: Number, reflect: true }) minchars = 3;
```

Comment:
I don't like this name. "minchars" of what? Seeing this on a SAYT tag says to me that something displayed in the SAYT component should have at minimum 3 characters. At least, that's the case for me. I think "min" and "length" should be in the name, but "minlength" has the same problem. @geoffcoutts suggested "minSearchLength" or "minTermLength"; it's still not great, but it's better.

As for why you shouldn't use "chars", a "character" can be tricky to define. Take the following two e's: é and é. They look identical and like single characters, but you'll find that the two actually have different lengths:

```js
console.log('é'.length); // 1
console.log('é'.length); // 2
```

This is because the first one is a combined character, whereas the second is a character with a combining diacritic. "Length" is more predictable as it tends to deal with the number of code points.